packages/oui-dual-list/dual-list.less

```diff
@@ -0,0 +1,209 @@
+  }
+
+  &__toggle-buttons {
+    margin-left: auto;
+    border: none;
+    background-color: inherit;
+    .oui-icon {
+      font-size: @oui-dual-list-icon-size;
+    }
+    @media screen and (min-width: @oui-responsive-breakpoint-phone) {
+      display: none;
+    }
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    padding: 0;
+    border-radius: 0 0 rem-calc(3px) rem-calc(3px);
+    min-height: 0;
+  }
+
+  &__content-body {
+    display: flex;
+    flex-direction: column;
+    min-height: 0;
+
+    & > .oui-dual-list__filter > form {
+      margin: 0;
+    }
+  }
+
+  &__content-empty {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex: 1;
+
+    p {
+      color: @oui-color-dual-list-empty-label;
+    }
+  }
+
+  &__target > &__content {
+    @media screen and (min-width: @oui-responsive-breakpoint-phone) {
+      display: flex;
+    }
+  }
+
+  &__filter {
+    position: relative;
+    padding: rem-calc(10px) @oui-dual-list-left-padding;
+
+    & > input {
+      background: @oui-color-dual-list-background;
+      border: 1px solid @oui-color-dual-list-header-background;
+      box-shadow: inset rem-calc(2px) rem-calc(2px) 1px #e9ebe7;
+      border-radius: rem-calc(3px);
+      width: 100%;
+      padding-right: 3rem;
+      height: rem-calc(40px);
+      padding-left: rem-calc(10px);
+    }
+
+    & > .oui-icon {
+      position: absolute;
+      right: @oui-dual-list-left-padding + rem-calc(10px);
+      top: 50%;
+      transform: translateY(-50%);
+      font-size: 2rem;
+    }
+  }
+
+  &__add-all-link, &__remove-all-link {
+    padding-left: @oui-dual-list-left-padding;
+    text-decoration: underline;
+    color: @oui-color-dual-list-link;
+    &:hover {
+      cursor: pointer;
+    }
+    &:hover, &:active {
+      color: @oui-color-dual-list-link;
+    }
+
+    & button {
+      padding: 0;
+      margin: 0;
+    }
+  }
+
+  &__remove-all-link {
+    @media screen and (min-width: @oui-responsive-breakpoint-phone) {
+      padding: rem-calc(5px) @oui-dual-list-left-padding;
+      margin-left: auto;
+    }
+  }
+
+  &__add-all-link {
+    @media screen and (min-width: @oui-responsive-breakpoint-phone) {
+      padding-left: @oui-dual-list-left-padding + @oui-dual-list-left-padding;
+    }
+  }
+
+  &__list {
+    list-style: none;
+    padding: 0;
+    overflow: auto;
+    margin: 0;
+
+    @media screen and (min-width: @oui-responsive-breakpoint-phone) {
+      padding: 0 @oui-dual-list-left-padding;
+    }
+  }
+
+  &__item {
+    padding: rem-calc(6px) @oui-dual-list-right-padding;
+    margin-bottom: rem-calc(2px);
+    background-color: @oui-color-dual-list-background;
+    position: relative;
+
+    @media screen and (min-width: @oui-responsive-breakpoint-phone) {
+      margin-bottom: rem-calc(5px);
+    }
+
+    &_text {
+      display: block;
+      width: 85%;
+    }
+
+    &_add-icon, &_remove-icon, &_loading-icon, &_add-label {
```

Comment:
Same here, not modifier :)

--`&_`
++`&-`

Reply:
it's not a modifier but its an element. The element is separated by - right?

Reply:
complete class name is like "oui-dual-list__item_loading-icon"

Reply:
`oui-dual-list__item_loading-icon` is not a modifier of `oui-dual-list__item` but an element of it.
To be compliant, it should be `oui-dual-list-item__loading-icon` or `oui-dual-list__item-loading-icon`.

See for example `class="oui-button oui-button_primary"`